File: duplicate-assert/java-junit/MasterClientTest_openCloseTest.java
public class MasterClientTest {
  @Test
  public void openCloseTest() throws FileAlreadyExistException, InvalidPathException, TException,
      IOException {
    MasterClient masterClient = new MasterClient(mMasterInfo.getMasterAddress(), mExecutorService);
    Assert.assertFalse(masterClient.isConnected());
    masterClient.connect();
    Assert.assertTrue(masterClient.isConnected());
    masterClient.user_createFile("/file", "", Constants.DEFAULT_BLOCK_SIZE_BYTE, true);
    Assert.assertTrue(masterClient.getFileStatus(-1, "/file") != null);
    masterClient.close();
    Assert.assertFalse(masterClient.isConnected());
    masterClient.connect();
    Assert.assertTrue(masterClient.isConnected());
    Assert.assertTrue(masterClient.getFileStatus(-1, "/file") != null);
  }
}

File: smelly/duplicate-assert/CSharpXUnit/CSharpXUnit.Tests/Tests/MasterClientTest_openCloseTest.cs
public class MasterClientTest
{
    [Fact]
    public void OpenCloseTest()
    {
        var masterClient = new MasterClient(mMasterInfo.GetMasterAddress(), mExecutorService);
        Assert.False(masterClient.IsConnected());
        masterClient.Connect();
        Assert.True(masterClient.IsConnected());
        masterClient.UserCreateFile("/file", "", Constants.DEFAULT_BLOCK_SIZE_BYTE, true);
        Assert.NotNull(masterClient.GetFileStatus(-1, "/file"));
        masterClient.Close();
        Assert.False(masterClient.IsConnected());
        masterClient.Connect();
        Assert.True(masterClient.IsConnected());
        Assert.NotNull(masterClient.GetFileStatus(-1, "/file"));
    }
}
